pkg/genericapiserver/options/server_run_options.go

```diff
@@ -159,6 +159,7 @@ func NewServerRunOptions() *ServerRunOptions {
 func (o *ServerRunOptions) WithEtcdOptions() *ServerRunOptions {
 	o.StorageConfig = storagebackend.Config{
 		Prefix: DefaultEtcdPathPrefix,
+		Quorum: DefaultReadQuorumFlag,
```

Comment:
Why do you think it will slow down single server case? If this flag redirects read to the leader, in single server cluster it should be a no-op.